token/context.go

```diff
@@ -0,0 +1,43 @@
+package token
+
```

Comment:
I guess, fabric8-common is not ready to be consumed yet, so copied context.go and context_test.go files from fabric8-tenant project.